liferay-frontend/liferay-portal
# LPS-116361 Adds Admin portlet for Remote Apps

Originally https://github.com/liferay-frontend/liferay-portal/pull/200
And then broken down as requested: https://github.com/brianchandotcom/liferay-portal/pull/92278#issuecomment-669475551
Part 1: https://github.com/liferay-frontend/liferay-portal/pull/217
Part 2: This PR

To test it:
1. Go to `Remote Apps ` under `Custom Apps` in the `Applications Menu`
![Screen Shot 2020-08-11 at 09 08 20](https://user-images.githubusercontent.com/156388/89895692-9ba27b80-dbb2-11ea-9e53-85a0e32d5cd0.png)

2. Click the plus button and add a new Remote App
![Screen Shot 2020-08-11 at 09 08 59](https://user-images.githubusercontent.com/156388/89895715-a8bf6a80-dbb2-11ea-9280-476a61b3ddbf.png)

![Screen Shot 2020-08-11 at 09 08 34](https://user-images.githubusercontent.com/156388/89895761-bbd23a80-dbb2-11ea-8808-ab48211a451a.png)

3. The newly created app should now be available as a widget to be added on a page
![Screen Shot 2020-08-11 at 09 09 42](https://user-images.githubusercontent.com/156388/89895824-d1dffb00-dbb2-11ea-8b52-26955ba13af2.png)

4. When added, an iframe pointing to the app url should be rendered
![Screen Shot 2020-08-11 at 09 13 30](https://user-images.githubusercontent.com/156388/89895911-f2a85080-dbb2-11ea-8fb0-3375f350ba8c.png)
